packages/next/src/index.ts

```diff
@@ -1927,8 +1927,8 @@ export const build: BuildV2 = async ({
             const groupPageKeys = Object.keys(group.pages);
 
             const launcher = launcherData.replace(
-              /\/\/ __LAUNCHER_PAGE_HANDLER__/g,
-              `
+              'let page = {};',
```

Comment:
The `// __LAUNCHER_PAGE_HANDLER__` comment was getting stripped away by esbuild, so changed the string replacement to be `let page = {};`.